Content/FeatureComparison.md

```diff
@@ -1,28 +1,32 @@
+| ARM64 support                                                      | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No            |                  | ![Orange Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff9900&text=+) Trial | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
+| [RAM Disk integration](../PlusContent/RamDiskSupport.md)           | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No  | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes (no UI) |                  | ![Red Color](https://placeholder.antonshell.me/img?width=15&color_bg=ff0000&text=+) No       | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                        | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes                | ![Green Color](https://placeholder.antonshell.me/img?width=15&color_bg=00ff00&text=+) Yes |
+
+\*) A personal type certificate once expired does not unlock features in builds compiled after it expired, builds compiled before that time retain their exclusive unlocked features state.
+\*\*) Personal covers all devices you use yourself, so if you have a laptop and a desktop, one certificate covers both.
+\*\*\*) A family pack can be used for the entire family, it is not required for the family to live in the same household, so the certificate can be used for children who have moved out or your grandparents.
+\*\*\*\*) For as long as the service exists.
```

Comment:
A new line should be added between the lines, otherwise it looks like a single line.